battleship-server/src/main/java/com.epam.solid.nie/server/server/Player.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.solid.nie.server.server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public interface Player {
+    void register(ServerSocket serverSocket) throws IOException;
+
+    void inform(String s) throws IOException;
```

Comment:
hmm IMO inform isn't the best choice for method in player interface. Maybe try to rename it.

Reply:
any suggestion?